fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider.java

```diff
@@ -116,6 +125,34 @@ protected FabricTagBuilder getOrCreateTagBuilder(TagKey<T> tag) {
 		return new FabricTagBuilder(super.getOrCreateTagBuilder(tag));
 	}
 
+	/**
+	 * Gets an {@link AliasGroupBuilder} with the given ID.
+	 *
+	 * @param groupId the group ID
+	 * @return the alias group builder
+	 */
+	protected AliasGroupBuilder getOrCreateAliasGroupBuilder(Identifier groupId) {
```

Comment:
I always wanted to drop the `getOrCreate`, but haven't gotten an opportunity to do so. Maybe do that for future ones?

Reply:
Sure, I just have it to match the existing methods. I think `aliasGroup` is probably enough (the "builder" part is also not very useful)